src/buffer.c

```diff
@@ -878,60 +878,6 @@ reset_buffer_local_variables (struct buffer *b, bool permanent_too)
     }
 }
 
```

Comment:
You omitted the comment below. It still sounds useful.

Reply:
Done.